app/views/internal/profile_fields/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="crayons-card grid p-6 mb-6 gap-1">
+  <%= form_for [:internal, ProfileField.new], html: { class: "inline-form" } do |form| %>
+    <div class="form-group">
```

Comment:
**nitpick (non-blocking):** The indentation for this `div` seems off. OCD: triggered 😉